src/ray/stats/metric_exporter.h

```diff
@@ -0,0 +1,88 @@
+// Copyright 2017 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RAY_METRIC_EXPORTER_H
+#define RAY_METRIC_EXPORTER_H
+#include "absl/memory/memory.h"
+#include "opencensus/stats/stats.h"
+#include "opencensus/tags/tag_key.h"
+
+#include "ray/stats/metric.h"
+#include "ray/stats/metric_exporter_client.h"
+#include "ray/util/logging.h"
+#include "ray/util/util.h"
+
+namespace ray {
+namespace stats {
+
+/// Main function of metric exporter is collecting indicator information from
+/// opencensus data view, and finally sends it to the remote ( for example
+/// send metrics to gcs service through rpc. How to use it? Register metrics
+/// exporter after raylet main thread launched. Actually its exporter
```

Comment:
```suggestion
/// exporter after a main thread launched. 
```

This library will be used for `core_worker` and `gcs_server`, so I think it is unnecessary to mention raylet here.